week-3/D-array-filter/exercise.js

```diff
@@ -8,7 +8,13 @@
 
 var pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
 
-var pairsByIndex; // Complete this statement
+var pairsByIndex = pairsByIndexRaw.filter(function(indexes) {
+  if (indexes !== null || indexes !== "string" || indexes !== "Boolean") {
+    return indexes;
```

Comment:
This is close, but not exactly right. Remember that the filter method expects you to return a boolean (`true` or `false`) from the callback function. Here we're returning the element itself.